queries.sql

```diff
@@ -6,3 +6,60 @@ SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered;
 SELECT * FROM animals WHERE name != 'Gabumon';
 SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+----------------------------------------------------------------
+
+BEGIN;
+UPDATE animals
+SET species = 'unspecified';
+
+ROLLBACK;
+
```

Comment:
- [ ] @Salwa99 great work on your query just like in your screenshot kindly add the `SELECT * from animals`, before and after rollback to showcase the changes
![image](https://user-images.githubusercontent.com/61408860/222196787-5951ab52-53c0-4e22-b03c-747813b0fd94.png)